src/components/AddExecomDialog.js

```diff
@@ -0,0 +1,353 @@
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    FormControl,
+    FormControlLabel,
+    InputLabel,
+    makeStyles,
+    MenuItem,
+    Select,
+    Snackbar,
+    TextField,
+    Grid,
+    InputAdornment,
+    Input,
+    Checkbox,
+    OutlinedInput,
+    // Typography,
+  } from '@material-ui/core';
+  import axios from 'axios';
+  import React, { useState, useEffect } from 'react';
+  import {hostname,societyNames} from '../links';
+  import { useImageUploader,FileUploadButton } from '../utils';
+
+  import Alert from '@material-ui/lab/Alert';
+  import { CircularProgress } from '@material-ui/core';
+
+  const useStyles = makeStyles(theme => ({
+    root: {
+      position: 'absolute',
+      top: '30%',
+    },
+    formControl: {
+      minWidth: 240,
+    },
+    fields: {
+      padding: theme.spacing(1),
+      '& .MuiOutlinedInput-root': {
+        '& fieldset': {
+          borderColor: 'green',
+        },
+        '&:hover fieldset': {
+          border:"2px solid",
+          borderColor: 'green',
+        },
+        '&.Mui-focused fieldset': {
+          border: '2px solid green', 
+        },
+      },
+    },
+  }));
+
+  //Nice prototype to convert Date objects to datetime local strings
+// eslint-disable-next-line
+Date.prototype.toDatetimeLocal = function toDatetimeLocal() {
+  var date = this,
+    ten = function (i) {
+      return (i < 10 ? '0' : '') + i;
+    },
+    YYYY = date.getFullYear(),
+    MM = ten(date.getMonth() + 1),
+    DD = ten(date.getDate()),
+    HH = ten(date.getHours()),
+    II = ten(date.getMinutes()),
+    SS = ten(date.getSeconds());
+  return YYYY + '-' + MM + '-' + DD + 'T' + HH + ':' + II + ':' + SS;
+};
+  export const AddExecomDialog = props => {
+    const classes = useStyles();
+  
+    // Textfields related
+    const [values, setValues] = useState({
+      firstname: '',
+      lastname: '',
+      position:'',
+      sid:props.sid,
+      tenureStart:new Date().toDatetimeLocal(),
+      tenureEnd: '',
+      imagepath:'',
+    });
+
+
+    useEffect(() => {
+      if (props.data !== undefined) {
+        console.log("data")
+        setValues({
+          ...props.data,
+          tenureStart: props.data.tenureStart.toDatetimeLocal(),
```

Comment:
just a small suggestion use `convertDatesinValuestoUTC` defined in `utils.js` to convert `tenureStart` & `tenureEnd` for proper value (Don't think it will matter much but to be safe) _not on priority_